### PR TITLE
CORTX-31128 F-71A TE : Automate tests

### PR DESCRIPTION
**Pull request by RajeshPratapSInghChouhan**
# Problem Statement
- CORTX-31128 F-71A: TA : Automate tests
[TEST-41156](https://jts.seagate.com/browse/TEST-41156)
[TEST-41172](https://jts.seagate.com/browse/TEST-41172)
[TEST-41173](https://jts.seagate.com/browse/TEST-41173)

# Design
[TDS](https://seagate-systems.atlassian.net/wiki/spaces/PRIVATECOR/pages/980090952/QA+F-71A+Capacity+usage+User+Level)

# Coding
   Checklist for Author
-  [ ] Coding conventions are followed and code is consistent

# Testing 
  Checklist for Author
-  [ ] New/Affected tests are executed on Latest Build
-  [ ] Attach test execution logs
-  [ ] Collection tested and no collection error introduced (`pytest --local True --collect-only`)

# Review Checklist 
  Checklist for Author
- [ ] JIRA number/GitHub Issue added to PR
- [ ] PR is self reviewed
- [ ] Jira and state/status is updated and JIRA is updated with PR link
- [ ] Check if the description is clear and explained

# Impact Analysis
  Checklist for Author/Reviewer/GateKeeper
-  [ ] If change in any common function, make sure to update all calls and execute all affected tests.

# Documentation
  Checklist for Author
-  [ ] Changes done to ReadMe / WIKI / Confluence page / Quick Start Guide

### CODE REVIEW

File: tests/csm/rest/test_capacity_quota.py [hunk 1]
```diff
@@ -641,3 +642,227 @@ def test_40640(self):
...
+    @pytest.mark.parallel
+    @pytest.mark.tags('TEST-41172')
+    def test_41172(self):
+        """
+        Test get capacity usage stats API of multiple S3 users, and 1 bucket per user.
+        """
+        test_case_name = cortxlogging.get_frame()
+        self.log.info("##### Test started -  %s #####", test_case_name)
+        test_cfg = self.csm_conf["test_41172"]
+        max_size = test_cfg["max_size"]
```
Comment (kachhwahadivya) [inline tests/csm/rest/test_capacity_quota.py:660]: available size

File: tests/csm/rest/test_capacity_quota.py [hunk 2]
```diff
@@ -641,3 +642,227 @@ def test_40640(self):
...
+
+            obj_name_prefix = "created_obj"
+            obj_name = f'{obj_name_prefix}{time.perf_counter_ns()}'
+            random_size = self.csm_obj.random_gen.randrange(1, max_size)
+            num_objects = math.floor(max_size/random_size)
+            data_size = num_objects * random_size
+            self.log.info("Step 3: Create %s objects of Random size totals to %s bytes",
+                          num_objects, data_size)
+            for obj in range(0, num_objects):
+                resp = s3_misc.create_put_objects(obj_name, bucket,
```
Comment (kachhwahadivya) [inline tests/csm/rest/test_capacity_quota.py:695]: Can u add logging inside for loop?

Comment (RajeshPratapSInghChouhan) [reply tests/csm/rest/test_capacity_quota.py:696]: done

**Review by kachhwahadivya — APPROVED** [review]

**Comment by kachhwahadivya** [comment]
Collection log and pylint for changed files

**Review by ketanarlulkar — APPROVED** [review]

**Review by ravindrachoudhari — APPROVED** [review]

**Review by aditipande96 — APPROVED** [review]

**Comment by RajeshPratapSInghChouhan** [comment]
(virenv) [root@ssc-vm-g2-rhev4-1909 cortx-test]# pytest --local True --collect-only
:::::::::::::::::::::::::::::::::::::::::::::::::::::::
Successfully unmounted directory

 ============================2175 tests collected in 6.12s ============================

**Comment by RajeshPratapSInghChouhan** [comment]
# pylint tests/csm/rest/test_capacity_quota.py
************* Module csm.rest.test_capacity_quota
tests/csm/rest/test_capacity_quota.py:837:13: W0511: TODO: Error code and message check part (fixme)
tests/csm/rest/test_capacity_quota.py:877:12: W0511: TODO: Error code and message check part (fixme)
tests/csm/rest/test_capacity_quota.py:966:12: W0511: TODO: Error code and message check part (fixme)
tests/csm/rest/test_capacity_quota.py:974:12: W0511: TODO: Error code and message check part (fixme)
tests/csm/rest/test_capacity_quota.py:31:0: E0401: Unable to import 'commons' (import-error)
tests/csm/rest/test_capacity_quota.py:32:0: E0401: Unable to import 'commons' (import-error)
tests/csm/rest/test_capacity_quota.py:33:0: E0401: Unable to import 'commons.params' (import-error)
tests/csm/rest/test_capacity_quota.py:34:0: E0401: Unable to import 'commons.utils' (import-error)
tests/csm/rest/test_capacity_quota.py:35:0: E0401: Unable to import 'commons.utils' (import-error)
tests/csm/rest/test_capacity_quota.py:36:0: E0401: Unable to import 'config.s3' (import-error)
tests/csm/rest/test_capacity_quota.py:37:0: E0401: Unable to import 'libs.csm.csm_interface' (import-error)
tests/csm/rest/test_capacity_quota.py:38:0: E0401: Unable to import 'libs.s3' (import-error)
tests/csm/rest/test_capacity_quota.py:39:0: E0401: Unable to import 'libs.s3.s3_multipart_test_lib' (import-error)
tests/csm/rest/test_capacity_quota.py:40:0: E0401: Unable to import 'libs.s3.s3_test_lib' (import-error)
tests/csm/rest/test_capacity_quota.py:43:0: R0904: Too many public methods (24/20) (too-many-public-methods)

------------------------------------------------------------------
Your code has been rated at 9.33/10 (previous run: 9.31/10, +0.02)

**Pull request merged**